include/deal.II/fe/fe_q_bubbles.h

```diff
@@ -91,10 +99,16 @@ class FE_Q_Bubbles
   FE_Q_Bubbles(const unsigned int p);
 
   /**
-   * Constructor for tensor product polynomials with support points @p points
-   * plus bubble enrichments based on a one-dimensional quadrature formula.
-   * The degree of the finite element is <tt>points.size()</tt>. Note that the
-   * first point has to be 0 and the last one 1.
+   * Constructor for tensor product polynomials with support points @p
+   * points plus bubble enrichments based on a one-dimensional
```

Comment:
I think breaking the line here might be an issue for `doxygen`.
```suggestion
   * Constructor for tensor product polynomials with support points 
   * @p points plus bubble enrichments based on a one-dimensional
```